tools/api-client/python/lib/random_ai.py

```diff
@@ -1114,9 +1120,12 @@ def _is_valid_attack_of_type_Skill(self, attackers, defenders, non_attackers):
     contributions = []
     for non_attacker in non_attackers:
       if 'Fire' in non_attacker['skills']:
-        # non-participating fire dice can contribute from 0 to that die's min
+        # non-participating fire dice can contribute 0, or anywhere
+        # from the minimum amount by which the die can be turned
+        # down, to the amount that takes the die to its minimum valuerange_max = non_attacker['value'] - self._die_fire_min(non_attacker)
```

Comment:
Not a blocker, but this comment is garbled - it has the text of the following line appended to it.

Reply:
Ah, I was wondering where that line had gone!